glossary.md

```diff
@@ -88,6 +88,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | data manipulation       | thao tác với dữ liệu            | [https://git.io/Jvohh](https://git.io/Jvohh) |
 | deep learning           | học sâu                         |                                              |
 | deferred initialization | khởi tạo trễ                    | [https://git.io/Jfe1i](https://git.io/Jfe1i) |
+| Densely Connected Network | Mạng Kết nối Dày đặc          |                                              |
```

Comment:
có nên dịch luôn là `mạng kết nối đầy đủ` k nhỉ

Reply:
Em nghĩ sẽ dễ nhầm sang `Dense Layer`.

Reply:
Mạng kết nối đầy đủ là `fully-connected layer` rồi

Reply:
Yeah, thực ra mình ko thích gộp nhiều từ đồng nghĩa tiếng anh sang một từ tiếng việt duy nhất như kiểu kernel/filter lắm. Mình thấy cứ để đa dạng hay hơn.

Reply:
> Yeah, thực ra mình ko thích gộp nhiều từ đồng nghĩa tiếng anh sang một từ tiếng việt duy nhất như kiểu kernel/filter lắm. Mình thấy cứ để đa dạng hay hơn.

Me too!

Reply:
ớ thế sao trước mọi người k bảo luôn :)) vậy sửa lại nhé

Reply:
ơ giờ mình mới nhận ra `densely connected net` khác hoàn toàn với `fully-connected net` :)))
Sao tác giả lại bỏ chữ `convolutional` trong `densely connected convolutional net` nhỉ

Reply:
Em thấy google `densely connected net` vẫn ra kết quả cho `densely connected convolutional net`, nên tiếng Anh để thế chắc là ko sao. Còn dịch ra tiếng Việt thì nhập nhằng ở chỗ khi code thì `fully-connected layer` lại dùng `Dense()`, nên nếu dịch DenseNet là `mạng kết nối dày đặc` thì dễ nhầm. 
Chắc sửa lại thành `mạng tích chập kết nối dày đặc` cho đủ nghĩa.

Reply:
right, để mình update luôn